src/Components/Shared/src/ComponentParametersTypeCache.cs

```diff
@@ -25,6 +26,7 @@ internal class ComponentParametersTypeCache
             }
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026", Justification = "This pattern is not linker friendly.")]
```

Comment:
This shouldn't be suppressed. But instead have `RequiresUnreferencedCode` attribute.